packages/ui/src/components/atoms/Button.tsx

```diff
@@ -158,10 +159,10 @@ const ButtonStyle = styled.button<ButtonProps>`
 
   &:active {
     box-shadow: 0px 0px 4px 4px rgba(0, 0, 0, 0.4) inset;
-    transition: box-shadow ${buttonAnimationData.duration}
-      ${buttonAnimationData.curve};
+    transition: box-shadow ${buttonAnimationData.duration};
+    ${buttonAnimationData.curve};
   }
-
+  ${props => props.disabled && 'opacity: 0.5;'}
```

Comment:
Is this change required for all types of buttons?

Reply:
Done